aggregators/1.0/modules/ROOT/pages/aggregators-group-example.adoc

```diff
@@ -0,0 +1,160 @@
+= Group-Based Aggregator Example - Mule 4
+
+The Aggregators module *Group based aggregator* scope enables you to aggregate elements in different groups according to a group ID. The following example shows how to configure this scope to aggregate up to `3` HTTP request calls per group based on the group ID sent in the request. Then, the application transforms the calls via DataWeave into random ID numbers. For each group that completes the aggregation of `3` elements, the application logs all the aggregated random ID number elements of the group.
```

Comment:
```suggestion
The Aggregators module *Group based aggregator* scope enables you to aggregate elements into groups by group ID. The following example shows how to configure this scope to aggregate up to `3` HTTP request calls per group based on the group ID sent in the request. The application than transforms the calls via DataWeave into random ID numbers. For each group that completes the aggregation of `3` elements, the application logs all the aggregated random ID number elements of the group.
```
It's probably just me, but take another look to be sure that there is no confusion between the group ID sent in the request and the random ID "number elements" (?) of the group. I see that these are syntactically `GroupId` and `Source ID`; I wonder if we should adopt this naming into the narrative.